integration_test.go

```diff
@@ -0,0 +1,91 @@
+package gitquery_test
+
+import (
+	"context"
+	"sort"
+	"strings"
+	"testing"
+
+	"github.com/src-d/gitquery"
+	"github.com/src-d/gitquery/internal/function"
+	"github.com/stretchr/testify/require"
+	fixtures "gopkg.in/src-d/go-git-fixtures.v3"
+	sqle "gopkg.in/src-d/go-mysql-server.v0"
+	"gopkg.in/src-d/go-mysql-server.v0/sql"
+)
+
+type sortFunc func(i, j int) bool
+
+type rowSortFunc func([]sql.Row) sortFunc
+
+func TestIntegration(t *testing.T) {
+	engine := sqle.New()
+	require.NoError(t, fixtures.Init())
+	defer func() {
+		require.NoError(t, fixtures.Clean())
+	}()
+
+	path := fixtures.ByTag("worktree").One().Worktree().Root()
+	pool := gitquery.NewRepositoryPool()
+	_, err := pool.AddGit(path)
+	require.NoError(t, err)
+
+	engine.AddDatabase(gitquery.NewDatabase("foo", &pool))
+	function.Register(engine.Catalog)
+
+	testCases := []struct {
+		query    string
+		result   []sql.Row
+		sortFunc rowSortFunc
```

Comment:
Some queries, such as the aggregations can't be sorted using `ORDER BY`, so we need this to check some outputs :(

Reply:
Maybe you can use `require.ElementsMatch` instead of `require.Equal` to check the output